scalyr_agent/log_processing.py

```diff
@@ -2234,13 +2238,7 @@ def is_active(self):
     def set_inactive(self):
         self.__is_active = False
 
-    @property
-    def log_path(self):
-        """
-        @return:  The log file path
-        @rtype: str
-        """
-        # TODO:  Change this to just a regular property?
+    def get_log_path(self):
```

Comment:
We have to add more code to make Multiprocessing proxy objects work with properties, so it is easier just to use member functions.